patches/server/0793-Make-entity-tracker-use-highest-range-of-passengers.patch

```diff
@@ -0,0 +1,65 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Spottedleaf <Spottedleaf@users.noreply.github.com>
+Date: Sun, 7 Mar 2021 14:14:47 -0800
+Subject: [PATCH] Make entity tracker use highest range of passengers
+
+This should prevent people from having to up their animal range
+just so players riding horses or whatever can be seen at the
+configured player range.
+
+diff --git a/src/main/java/net/minecraft/server/level/ChunkMap.java b/src/main/java/net/minecraft/server/level/ChunkMap.java
+index da6553abafaa03e9324566d9632e4fee33d20b9b..7e86c4e9f92fece10e4ec23106386ae09d5a646a 100644
+--- a/src/main/java/net/minecraft/server/level/ChunkMap.java
++++ b/src/main/java/net/minecraft/server/level/ChunkMap.java
+@@ -222,7 +222,7 @@ public class ChunkMap extends ChunkStorage implements ChunkHolder.PlayerProvider
+     // inlined EnumMap, TrackingRange.TrackingRangeType
+     static final org.spigotmc.TrackingRange.TrackingRangeType[] TRACKING_RANGE_TYPES = org.spigotmc.TrackingRange.TrackingRangeType.values();
+     public final com.destroystokyo.paper.util.misc.PlayerAreaMap[] playerEntityTrackerTrackMaps;
+-    final int[] entityTrackerTrackRanges;
++    final int[] entityTrackerTrackRanges; public int getEntityTrackerRange(final int ordinal) { return this.entityTrackerTrackRanges[ordinal]; } // Paper - public read
```

Comment:
Is this some sort of Tuinity code style thing? Why is everything on the same line?